.github/workflows/online-sign.yml

```diff
@@ -0,0 +1,28 @@
+name: TUF-on-CI online signing
+
+permissions: {}
+
+on:
+  schedule:
+    - cron:  '17 1,7,13,19 * * *'
+  push:
+    branches: [ main ]
+    paths: ['metadata/**']
+  workflow_dispatch:
+
+jobs:
+  online-sign:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: 'write' # for OIDC identity access
+      contents: 'write' # for commiting snapshot/timestamp changes
+      actions: 'write' # for dispatching publish workflow
+
+    steps:
+      - id: online-sign
+        uses: theupdateframework/tuf-on-ci/actions/online-sign@moar-refactor
+        with:
+          force_publish: ${{ github.event_name != 'schedule' }}
```

Comment:
the action itself could probably figure out the triggering event... I'm not sure if it can but would be nice to not have it here